R/layer_image.R

```diff
@@ -36,6 +36,19 @@ ly_image <- function(fig, z, rows, cols, x = 0, y = 0, dw = 1, dh = 1,
     rows <- ncol(z)
     z <- array(z)
   }
+  
+  # palette checker / transformer from layer_hexbin minus function
+  if(is.character(palette)) {
```

Comment:
I'm wondering if the checking here should allow for a vector of colors.  We could check if is character and length is one, then it must be a palette name.  If length is greater than one, we can make sure it's a vector of valid color names (using `valid_color()`.  What do you think?